### ISSUE TITLE
Add `test_size` parameter to train_test_split method

### ISSUE THREAD

**Issue by iKintosh** [tinkoff-ai/etna-ts#175]
### 🚀 Feature Request

We can add `test_size` parameter because data in TSDataset are required to end at the same timestamp.

### Motivation

We should add ability to define size in test set as number of timestamps to use.

### Proposal

`test_size` and ANY timestamp options cannot be applied at the same time. In this case we should raise an error.

Use case:

```
train_test_split(test_size: int)
```

splits dataset in two, where test set has size equal to test_size.

### Checklist

- [X] Added feature request
- [X] Added motivation
- [X] Added proposal
